lib/web_ui/lib/src/engine/window.dart

```diff
@@ -58,7 +58,10 @@ class EngineFlutterWindow extends ui.SingletonFlutterWindow {
     });
   }
 
-  final Object _windowId;
+  final Object _viewId;
+
+  @override
+  Object get viewId => _viewId;
```

Comment:
Isn't this the same, but a little bit shorter?

```suggestion
  @override
  final Object viewId;
```